src/containers/AuthorsArticle/ArticleCard.js

```diff
@@ -6,7 +6,7 @@ const ArticleCard = props => (
     <li key={props.article.id} className="article">
       <div className="photo">
         <img
-          src={props.article.imgUrl || props.placeholderImg }
+          src={(props.article.imgUrl !== 'null' && props.article.imgUrl) ? props.article.imgUrl : props.placeholderImg }
```

Comment:
Checking null as a string because the value returned from the database as imgUrl when no image is null as a string.